samples/CommunityToolkit.Maui.Sample/Pages/Views/Maps/BasicMapsPage.xaml.cs

```diff
@@ -0,0 +1,17 @@
+using CommunityToolkit.Maui.Sample.ViewModels.Views;
+using Microsoft.Maui.Maps;
+
+namespace CommunityToolkit.Maui.Sample.Pages.Views;
+
+public partial class BasicMapsPage : BasePage<BasicMapsViewModel>
+{
+	public BasicMapsPage(BasicMapsViewModel mapsViewModel) : base(mapsViewModel) => InitializeComponent();
+
+	protected override void OnNavigatedTo(NavigatedToEventArgs args)
+	{
+		base.OnNavigatedTo(args);
+
+		basicMap.MoveToRegion(MapSpan.FromCenterAndRadius(new Location(50, 6), Distance.FromKilometers(1)));
```

Comment:
Weird enough, this _does_ seem to work where the pins sample page doesn't seem to work...?

Reply:
OnNavigatedTo is not a reliable thing. Moved everything on button clicked